funcx_websocket_service/server.py

```diff
@@ -160,5 +161,12 @@ async def handle_connection(self, ws, path):
 
     async def process_request(self, path, headers):
         if path == '/v2/health':
-            return http.HTTPStatus.OK, [], b"OK\n"
+            version_data = {
```

Comment:
@joshbryan-globus Hey Josh, can you check whether returning the version info would break the cloudwatch monitoring?

Reply:
It shouldn't.  We just check for status code right now, so as long as we return a 200 we are good.

Reply:
:+1: